doc/ArchInstall/arch-chroot_install.sh

```diff
@@ -39,10 +39,14 @@ EOF
 pacman -S --noconfirm --needed sudo
 
 # Setup users of the wheel group to be able to execute sudo commands with no password
-sed --in-place 's/^#\s*\(%wheel\s\+ALL=(ALL)\s\+NOPASSWD:\s\+ALL\)/\1/' /etc/sudoers
+echo "%wheel ALL=(ALL) ALL" > /etc/sudoers.d/wheel
+chown root:root /etc/sudoers.d/wheel
+chmod 0440 /etc/sudoers.d/wheel
 
 # Get sudo to insult users when they type a password wrong
 echo "Defaults insults" > /etc/sudoers.d/insults
+chown root:root /etc/sudoers.d/wheel
+chmod 0440 /etc/sudoers.d/insults
```

Comment:
This doesn't need to be in the arch install.
Although in general, the arch install needs to be stripped down